eclass/linux-mod-r1.eclass

```diff
@@ -471,6 +489,18 @@ linux-mod-r1_pkg_postinst() {
 	dist-kernel_compressed_module_cleanup "${EROOT}/lib/modules/${KV_FULL}"
 	_modules_update_depmod
 
+	if [[ -z ${ROOT} && ${MODULES_IUSE_INITRAMFS} ]] &&
+		use dist-kernel && use initramfs; then
+			dist-kernel_reinstall_initramfs "${KV_DIR}" "${KV_FULL}"
+	fi
```

Comment:
When breaking `if` in multiple lines, I prefer the `then` on a new line for less indenting ambiguity

aka:
```bash
	if [[ -z ${ROOT} && ${MODULES_IUSE_INITRAMFS} ]] &&
		use dist-kernel && use initramfs
	then
			dist-kernel_reinstall_initramfs "${KV_DIR}" "${KV_FULL}"
	fi
```